### PR TITLE
branding update for release/3.0 branch

### PR DESCRIPTION
**Pull request by vatsan-madhavan**
Update branding for release/3.0 branch to preview3

### CODE REVIEW

File: eng/common/templates/phases/publish-build-assets.yml [hunk 1]
```diff
@@ -28,12 +28,15 @@ phases:
             SecretsFilter: 'MaestroAccessToken'
           condition: ${{ parameters.condition }}
           continueOnError: ${{ parameters.continueOnError }}
-        - script: eng\common\publishbuildassets.cmd
-            /p:ManifestsPath='$(Build.StagingDirectory)/Download/AssetManifests'
-            /p:BuildAssetRegistryToken=$(MaestroAccessToken)
-            /p:MaestroApiEndpoint=https://maestro-prod.westus2.cloudapp.azure.com
-            /p:Configuration=$(_BuildConfig)
+        - task: PowerShell@2
```
Comment (stevenbrix) [inline eng/common/templates/phases/publish-build-assets.yml]: is this change intentional? looks like this is an arcade file

File: azure-pipelines.yml [hunk 2]
```diff
@@ -11,7 +11,8 @@ variables:
 
 # only trigger ci builds for the master branch
```
Comment (stevenbrix) [inline azure-pipelines.yml]: i would update this comment

File: eng/common/templates/phases/publish-build-assets.yml (hunk 1 above)
Comment (vatsan-madhavan) [reply eng/common/templates/phases/publish-build-assets.yml]: I just merged against master. Is this different from your copy of publish-build-assets.yml ?

Comment (vatsan-madhavan) [reply eng/common/templates/phases/publish-build-assets.yml]: I windiff'd with master - they are identical after this merge.
